dev-libs/libflatarray/libflatarray-0.3.0.ebuild

```diff
@@ -1,9 +1,9 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
```

Comment:
Please revbump when updating EAPI. When revbumping don't use `git mv` but make a new commit for bumping new ebuild, then new commit for removing old one(s).

Reply:
Okay, will do. Thanks!